src/app/ubs-admin/components/ubs-admin-table/ubs-admin-table.component.scss

```diff
@@ -52,22 +59,26 @@ input {
 
 .mat-row {
   min-height: 39px !important;
+  max-height: 39px !important;
```

Comment:
Can we avoid !important here?
Maybe we can use :host or :ng-deep instead?
https://blog.angular-university.io/angular-host-context/

If not, let it be :)

Reply:
If not, let it be :)